cmd/make_dbscale_sandbox.go

```diff
@@ -33,6 +33,13 @@ func exists(path string) (bool, error) {
 	return true, err
 }
 
+func check(e error) {
```

Comment:
check is defined in mysql.go, you can just invoke check.
Before that, you should change "check"  to "Check", which make the Check function public